main/res/values/strings.xml

```diff
@@ -1539,13 +1506,10 @@
 
     <!-- next things -->
     <string name="quote">To make geocaching easier, to make users lazier.</string>
-    <string name="powered_by">carnero</string>
     <!-- Note: Links here are just for appearance. See AboutActivity to make changes -->
     <string name="support_title">Support</string>
     <string name="website_title">Website</string>
-    <string name="facebook_title">Facebook</string>
     <string name="facebook_link"><a href="">c:geo page</a></string>
-    <string name="twitter_title">Twitter</string>
     <string name="market_title">Google Play</string>
```

Comment:
Maybe also Google Play?

Reply:
I had not included Google Play as I was unsure whether it has the same name in all languages. - Just cross checked with current translation files: At least in `values-ko` variant it differs (see string `market_title`).